enterprise/backup/src/main/java/org/neo4j/backup/HaBackup.java

```diff
@@ -160,12 +142,11 @@ private BackupOutcome fullBackup( FileSystemAbstraction fileSystem, String sourc
     {
         if ( !directoryIsEmpty( fileSystem, targetDirectory ) )
         {
-            throw new RuntimeException( "Can only perform a full backup into an empty directory but " +
-                    targetDirectory + " is not empty" );
+            throw new RuntimeException( "Can only perform a full backup into an empty directory but " + targetDirectory + " is not empty" );
         }
         long timestamp = System.currentTimeMillis();
         long lastCommittedTx = -1;
-        try ( PageCache pageCache = createPageCache( fileSystem, tuningConfiguration ) )
+        try ( PageCache pageCache = this.pageCacheOptional.orElse( createPageCache( fileSystem, tuningConfiguration ) ))
```

Comment:
I think checkstyle in 3.3. will complain about the lack of a space at the end, there.